packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.resolver.ts

```diff
@@ -22,6 +31,44 @@ export class ObjectMetadataResolver {
     private readonly beforeUpdateOneObject: BeforeUpdateOneObject<UpdateObjectPayload>,
   ) {}
 
+  private async resolveLabel(
```

Comment:
move this to field-metadata.service (should be a low level service to avoid circular dependecy)